ghzdac/correction.py

```diff
@@ -1165,10 +1188,17 @@ def DACifyFT(self, signal, t0=0, n=8192, offset=0, nfft=None, loop=False,
                 #decay times:
                 # add to qubit registry the following keys:
                 # settlingAmplitudes=[-0.05]  #relative amplitude
-                # settlingRates = [0.012]    #rate is in GHz, (1/ns)
+                # settlingRates = [0.01]    #rate is in GHz, (1/ns)
                 if np.alen(decayRates):
-                    freqs = 2j*np.pi*freqs
```

Comment:
seems like it would still be nice to precompute this; maybe something like:
```python
if np.alen(decayRates) or np.alen(reflectionRates):
    i_pi_freqs = 2j*np.pi*freqs
    if np.alen(decayRates):
        precalc /= ...

    if np.alen(reflectionRates):
        for rate, amplitude in zip(reflectionRates, reflectionAmplitudes):
            precalc /= (1.0 - amplitude) / (1.0 - amplitude * np.exp(-i_pi_freqs / rate))
```